lib/src/breakpoints_utils.dart

```diff
@@ -1,40 +1,65 @@
-import 'package:adaptive_breakpoints/adaptive_breakpoints.dart';
-
 import 'package:flutter/material.dart';
+import 'package:flutter_adaptive_scaffold/flutter_adaptive_scaffold.dart';
 
-extension Breakpoints on BuildContext {
-  /// All the breakpoints according to Material Design's breakpoint system.
-  BreakpointSystemEntry get breakpoints => getBreakpointEntry(this);
-
-  /// Double the default gutter size.
-  double get gutterLarge => (_material3Value ?? breakpoints.gutter) * 2;
-
-  /// The gutter size according to Material Design's breakpoints system.
-  double get gutter => _material3Value ?? breakpoints.gutter;
-
-  /// Half the default gutter size.
-  double get gutterSmall => (_material3Value ?? breakpoints.gutter) / 2;
-
-  /// A quarter the default gutter size.
-  double get gutterTiny => (_material3Value ?? breakpoints.gutter) / 4;
+import '../flutter_gutter.dart';
 
+/// A gap of the standard gutter size according to Material Design's breakpoints
+extension BreakpointExtensions on BuildContext {
   /// The margin size according to Material Design's breakpoints system.
-  double get margin => _material3Value ?? breakpoints.margin;
-
-  int get columns => breakpoints.columns;
+  double get margin => materialSpacing;
 
-  double? get _material3Value {
-    if (!Theme.of(this).useMaterial3) {
-      return null;
+  /// The size according to Material Design's with a specified [GutterType].
+  double gutter({GutterType type = GutterType.medium, double? scaleFactor}) {
+    switch (type) {
+      case GutterType.tiny:
+        return materialSpacing / (scaleFactor ?? 4);
+      case GutterType.small:
+        return materialSpacing / (scaleFactor ?? 2);
+      case GutterType.medium:
+        return materialSpacing;
+      case GutterType.large:
+        return materialSpacing * (scaleFactor ?? 2);
+      case GutterType.extraLarge:
+        return materialSpacing * (scaleFactor ?? 4);
+      case GutterType.expand:
+        return double.infinity;
     }
-    final Size size = MediaQuery.of(this).size;
-    if (size.width < 600 || size.height < 600) {
+  }
+
+  /// The size according to Material Design's breakpoints system.
+  double get materialSpacing {
+    if (Breakpoints.small.isActive(this)) {
       // Use the compact screen values.
       // See: https://m3.material.io/foundations/layout/applying-layout/compact
       return 16;
```

Comment:
Does the new `adaptive_scaffold` package no longer define these constants itself?

Speaking of, one design nit: I'd prefer the exact values be global constants (admittedly I should've done this myself in the previous version and was just too lazy)
We could make the values public or private, and in global namespace or just as static attributes, I guess I don't have strong feelings.

Reply:
No they changed all of that, and the breakpoints are not available either.